docker/alpine.Dockerfile

```diff
@@ -18,14 +18,31 @@ RUN apk add \
 
 ARG RELAY=v0.6.6
 
+RUN set -eux \
+  && apk add --no-cache --virtual .build-deps \
+  gcc \
+  make \
+  musl-dev \
+  openssl-dev \
+  # Build and install hiredis (>= 1.1.0) with SSL support \
+  && curl -L https://github.com/redis/hiredis/archive/refs/tags/v1.2.0.tar.gz | tar -xz \
```

Comment:
Nice to see that you put the _glue_ at the front of the line.
Now it causes no brain damage 🧠